doc/nrf/libraries/others/network_core_monitor.rst

```diff
@@ -0,0 +1,119 @@
+.. _network_core_monitor:
+
+Network core monitor
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This library monitors the network core status of the nRF5340 processor.
+
+Overview
+********
+
+It detects suspensions and resets in the network processor.
+When a reset occurs, you can read the cause of the reset.
+
+Implementation
+==============
+
+The library uses two general-purpose registers of the IPC peripheral in the application core and writes the state of the network processor to them.
+
+.. figure:: images/ncm_register.svg
+   :alt: Application of IPC peripheral registers
+
+   IPC peripheral registers
+
+The ``GPMEM[0]`` register is divided into two 16-bit parts.
+
+The ``COUNTER`` value is incremented by the network core every :kconfig:option:`CONFIG_NCM_APP_FEEDING_INTERVAL_MSEC`.
```

Comment:
```suggestion
The ``COUNTER`` value (in milliseconds) is incremented by the network core with the same value that is originally set in the :kconfig:option:`CONFIG_NCM_APP_FEEDING_INTERVAL_MSEC` Kconfig option.
```
I would add more details here - is the sentence still correct?

Reply:
`COUNTER` is not in milliseconds. 
This is a counter incremented by 1 every `CONFIG_NCM_APP_FEEDING_INTERVAL_MSEC` millisecond.

Reply:
I see, thanks. If we remove the incorrect part in brackets, would it be sufficient?

Reply:
After the off-line discussion, this sentence will remain unchanged.